modules/ocf_decal/manifests/website.pp

```diff
@@ -0,0 +1,43 @@
+class ocf_decal::website {
+
+  file {
+    ['/srv/www', '/srv/www/decal']:
+      ensure => directory,
+      owner  => ocfdecal,
+      group  => www-data,
+      mode   => '0755',
+      require => User['ocfdecal'];
+  }
+
+  apache::vhost { 'decal-redirect':
+    servername => 'decal.ocf.berkeley.edu',
+    serveraliases => [
+      'decal.ocf.io',
+      'decal.xcf.sh',
+      'decal.xcf.berkeley.edu'
+    ],
+    port => 80,
+    docroot => '/srv/www/decal',
+    redirect_status => 'permanent',
+    redirect_dest   => 'https://decal.ocf.berkeley.edu',
+  }
+
+  apache::vhost { 'decal-ssl':
+    servername => 'decal.ocf.berkeley.edu',
+    serveraliases => [
+      'decal.ocf.io',
+      'decal.xcf.sh',
+      'decal.xcf.berkeley.edu',
```

Comment:
I think these should all be redirecting too to the canonical domain unless you want all of these to work too.

Reply:
I think it should be ok, the short URLs are kinda nice